cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc

```diff
@@ -0,0 +1,33 @@
+[id='viewing-pipeline-logs-using-the-openshift-logging-operator']
+= Viewing pipeline logs using the OpenShift Logging Operator
+
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: viewing-pipeline-logs-using-the-openshift-logging-operator
+
+toc::[]
+
+The logs generated by pipeline runs, task runs, and event listeners are stored in their respective pods. It is useful to review and analyze logs for troubleshooting and audits.
+
+However, retaining the pods indefinitely leads to unnecessary resource consumption and cluttered namespaces.
+
+To eliminate any dependency on the pods for viewing pipeline logs, you can use the OpenShift Elasticsearch Operator and the OpenShift Logging Operator. These Operators help you to view pipeline logs by using the link:https://www.elastic.co/guide/en/kibana/6.8/connect-to-elasticsearch.html[Elasticsearch Kibana] stack, even after you have deleted the pods that contained the logs.
+
+[id="prerequisites_{context}"]
```

Comment:
```suggestion
[id="prerequisites_viewing-pipeline-logs-using-the-openshift-logging-operator"]
```
See guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies